peer.go

```diff
@@ -1952,6 +1960,13 @@ func (p *peer) handleLinkFailure(failure linkFailureReport) {
 		}
 	}
 
+	// If this is a permanent failure, we will mark the channel borked.
```

Comment:
Should we add a sanity check to make sure both `ForceClose` and `PermanentFailure` are not set?

Reply:
Not sure if they will always be mutually exclusive? 

Added a logging statement: https://github.com/lightningnetwork/lnd/pull/2018/commits/03f38071e6a68dd98def37746983173f0cbb7260

Reply:
in the process of force closing a channel, we will already mark it as borked. should we skip this if `ForceClose` is true as well?